deploy/kubedirector/kubedirector.hpe.com_kubedirectorconfigs_crd.yaml

```diff
@@ -46,6 +46,8 @@ spec:
               type: string
             clusterSvcDomainBase:
               type: string
+            defaultNamingScheme:
+              type: boolean
```

Comment:
I think we want to leave ourselves some wiggle room for future changes with this configurability... rather than make it a boolean switch, it should probably be an enum, handled similarly to defaultServiceType. (This applies to the kdcluster-level property too of course.)

As for what the allowed values of the enum should be... well, for now there are two. For "old style" maybe call it something like "UID" or "Generate"; for "new style" maybe "ClusterAndRole".

Also, in order to be backwards compatible, the default value for this top-level config property should be the one for old style, so that behavior is the same in new KD releases unless someone explicitly chooses otherwise.